### PR TITLE
Expose mapping search task to Search Kit

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
Expose Contact Mapping task to Search Kit

Before
----------------------------------------
No Mapping task

After
----------------------------------------
Mapping task option

Technical Details
----------------------------------------
This is also a work in progress but it at least gets it show in the list of search tasks, the issue I am currently running into is that at least with Open Street Maps is that when the popup opens this code https://github.com/civicrm/civicrm-core/blob/master/templates/CRM/Contact/Form/Task/Map/OpenStreetMaps.tpl#L156 doesn't seem to fire correctly so initmap doesn't get run not sure what the correct change should be

ping @colemanw @eileenmcnaughton @JoeMurray

### CODE REVIEW

File: CRM/Contact/Form/Task.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ public static function preProcessCommon(&$form) {
     $form->_contactIds = [];
     $form->_contactTypes = [];
 
-    $isStandAlone = in_array('task', $form->urlPath) || in_array('standalone', $form->urlPath);
+    $isStandAlone = in_array('task', $form->urlPath) || in_array('standalone', $form->urlPath) || in_array('map', $form->urlPath);
```
Comment (eileenmcnaughton) [inline CRM/Contact/Form/Task.php:89]: I think we could call $form->isStandAlone() & then on the CRM_Contact_Form_Task_Map form it could return TRUE

Comment (seamuslee001) [reply CRM/Contact/Form/Task.php:89]: @eileenmcnaughton is that a new function or just something I haven't come across but yeh I'm thinking that might be what is needed, the map task does get used in a number of different contexts so will need to check if this does anything to them (don't think so but maybe)

**Comment by seamuslee001** [comment]
@eileenmcnaughton @JoeMurray @colemanw the display task of Mapping Contacts now works, I have tested as best as I can using both OSM and also Google Maps and both seem to work now. The only thing would be about the isStandAlone question IMO

**Comment by demeritcowboy** [comment]
Works for me for google. It uses a popup even if you have popups turned off but that's not a blocker. Regular search + map still works.

I don't have any comment on the isStandalone.

**Comment by eileenmcnaughton** [comment]
OK merging based on @demeritcowboy's testing - I think that standalone thing is still a bit of a mess but I won't make it a blocker on this PR

**Comment by colemanw** [comment]
Cool.

**Pull request merged**